seo/README.md

```diff
@@ -0,0 +1,14 @@
+# SEO Coding Culture
+
+## Overview
+
+This is a collection of documents that outline the SEO development culture & expectations.
+
+## Contents
+
+### Best practices and Principles
+
+ * [PR Best Practices](../pr-best-practices.md)
```

Comment:
We can remove this link, as it duplicates the main PR best practices link in the main TOC